pkg/virtctl/vnc/vnc.go

```diff
@@ -155,13 +168,37 @@ func (o *VNC) Run(cmd *cobra.Command, args []string) error {
 			glog.Infof("remote-viewer commandline: %v", args)
 		}
 
-		cmnd := exec.Command("remote-viewer", args...)
+		vncBin := ""
+		osType := runtime.GOOS
+		switch osType {
+		case "darwin":
+			if _, err := os.Stat(TIGER_VNC); os.IsExist(err) {
```

Comment:
Do not use anti-pattern
https://gist.github.com/mastef/05f46d3ab2f5ed6a6787#file-isexist_vs_isnotexist-go-L33

Reply:
To name the issue: `os.IsExist(err)` is meant to be used when you e.g. want to create a file and the file already exists. In your case you err will be `nil` in case the file exists.